src/EventListener/PimcoreAdminListener.php

```diff
@@ -0,0 +1,63 @@
+<?php declare(strict_types=1);
+
+namespace Neusta\PimcoreImportExportBundle\EventListener;
+
+use Pimcore\Event\BundleManager\PathsEvent;
+
+class PimcoreAdminListener
+{
+    public function addEditmodeCSSFiles(PathsEvent $event): void
+    {
+        $event->setPaths(array_merge(
+            $event->getPaths(),
+            [
+                $this->overloadFile('css/trinity.css', 'css/frontend.css'),
+                '/bundles/neustatrinitypimcore/css/editmode-style.css',
+            ],
+        ));
+    }
+
+    public function addCSSFiles(PathsEvent $event): void
+    {
+        $event->setPaths(array_merge(
+            $event->getPaths(),
+            [
+                '/bundles/neustatrinitypimcore/css/admin-gui-style.css',
+            ],
+        ));
+    }
+
+    public function addEditmodeJSFiles(PathsEvent $event): void
+    {
+        $event->setPaths(array_merge(
+            $event->getPaths(),
+            [
+                '/bundles/neustatrinitypimcore/js/pimcore/flickity.pkgd.min.js',
+                '/bundles/neustatrinitypimcore/js/pimcore/document/editables/markdown.js',
+                $this->overloadFile('js/trinity.umd.min.js', 'js/trinity-frontend.js'),
+            ],
+        ));
+    }
+
+    public function addJSFiles(PathsEvent $event): void
+    {
+        $event->setPaths(array_merge(
+            $event->getPaths(),
+            [
+                '/bundles/neustatrinitypimcore/js/startup.js',
+                '/bundles/neustatrinitypimcore/js/exportPage.js',
+            ],
+        ));
```

Comment:
```suggestion
        $event->addPaths([
            '/bundles/neustapimcoreimportexport/js/exportPage.js',
        ]);
```

Reply:
I've made a lot more changes to the Listener because most of the things we do not need and the directory naming was incorrect, too.

Reply:
But that doesn't mean we don't want this change, too.